libs/liblobby/lobby/lobby.lua

```diff
@@ -596,8 +596,17 @@ function Lobby:_OnUnfriend(userName)
 	self:_CallListeners("OnUnfriend", userName)
 end
 
-function Lobby:_OnFriendList()
-	self:_CallListeners("OnFriendList", self:GetFriends())
+function Lobby:_OnFriendList(friends)
+	self.friends = friends
+	self.friendCount = #friends
+
+	for _, userName in pairs(self.friendRequests) do
```

Comment:
I'm surprised you're updating this based on `self.friendRequests`, rather than `self.friends`, is this intentional?

Reply:
Not intentional, fixed :)